# Type: feat
# Subtype: Ex
# English Summary: Set the focusability of descendants to FOCUS_AFTER_DESCENDANTS in the constructor of DrawerLayout
# Chinese Summary: 在DrawerLayout的构造函数中设置后代的焦点可用性为FOCUS_AFTER_DESCENDANTS
# Entry Number: 26

# === Git Diff Content ===

diff --git a/v4/java/android/support/v4/widget/DrawerLayout.java b/v4/java/android/support/v4/widget/DrawerLayout.java
index e3e596535..bab235b5a 100644
--- a/v4/java/android/support/v4/widget/DrawerLayout.java
+++ b/v4/java/android/support/v4/widget/DrawerLayout.java
@@ -258,7 +258,7 @@ public class DrawerLayout extends ViewGroup {
 
     public DrawerLayout(Context context, AttributeSet attrs, int defStyle) {
         super(context, attrs, defStyle);
-
+        setDescendantFocusability(ViewGroup.FOCUS_AFTER_DESCENDANTS);
         final float density = getResources().getDisplayMetrics().density;
         mMinDrawerMargin = (int) (MIN_DRAWER_MARGIN * density + 0.5f);
         final float minVel = MIN_FLING_VELOCITY * density;
